workers/unity/Assets/Tests/EditmodeTests/Correctness/Core/WorkerFlagLoggingTests.cs

```diff
@@ -0,0 +1,78 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.TestUtils;
+using NUnit.Framework;
+
+namespace Improbable.Gdk.EditmodeTests
+{
+    [TestFixture]
+    public class WorkerFlagLoggingTests : MockBase
+    {
+        private const string ProtocolLoggingFlag = "protocol_logging_enabled";
+        private const long EntityId = 1;
+
+        private static EntityTemplate GetTemplate()
+        {
+            var template = new EntityTemplate();
+            template.AddComponent(new Position.Snapshot());
+            return template;
+        }
+
+        [Test]
+        public void Logging_disabled_when_ProtocolLoggingFlag_does_not_exist()
+        {
+            World
+                .Step(world => world.Connection.CreateEntity(EntityId, GetTemplate()))
```

Comment:
Why do these tests create entities?

Reply:
I somehow thought that was create connection😅